### PR TITLE
Updated deprecated and aliased keys in .travis.yml

### LINKED ISSUE #777: Update .travis.yml
## 🚀 Feature

According to travis, there are several warnings in the current .travis.yml configuration: https://travis-ci.org/pytorch/ignite/jobs/648860097/config?utm_medium=notification&utm_source=github_status
```
jobs.include.deploy: deprecated key skip_cleanup (not supported in dpl v2, use cleanup)
jobs.include.deploy: deprecated key skip_cleanup (not supported in dpl v2, use cleanup)
jobs.include.deploy: deprecated key skip_cleanup (not supported in dpl v2, use cleanup)
jobs.include.deploy: deprecated key skip_cleanup (not supported in dpl v2, use cleanup)
jobs.include.deploy: key github-token is an alias for token, using token
jobs.include.deploy: key skip-cleanup is not underscored, using skip_cleanup
root: missing os, using the default linux
jobs.include.deploy: key github-token is an alias for token, using token
jobs.include.deploy: key skip-cleanup is not underscored, using skip_cleanup
jobs.include.deploy: key keep-history is not underscored, using keep_history
jobs.include.deploy: key user is an alias for username, using username
jobs.include.deploy: key user is an alias for username, using username
jobs.include.deploy: key keep-history is not underscored, using keep_history
```

We need to improve this.

### PR DESCRIPTION
**Pull request by anmolsjoshi**
Fixes #777 

Description:
Updated deprecated and aliased keys in .travis.yml

[Travis CI config](https://travis-ci.org/pytorch/ignite/jobs/652835429/config?utm_medium=notification&utm_source=github_status) shows no warnings

Check list:
* [ ] New tests are added (if a new feature is added)
* [ ] New doc strings: description and/or example code are in RST format
* [ ] Documentation is updated (if required)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -112,16 +115,14 @@ jobs:
       # Deploy built docs : a) when PR is merged to master, b) when a tag is built
       deploy:
         - provider: pages
-          skip-cleanup: true
```
Comment (vfdev-5) [inline .travis.yml]: @anmolsjoshi shouldn't we still keep `skip_cleanup` as `cleanup` as suggested in the warning:
```
jobs.include.deploy: deprecated key skip_cleanup (not supported in dpl v2, use cleanup)
```

Comment (anmolsjoshi) [reply .travis.yml]: @vfdev-5 have a look [here](https://blog.travis-ci.com/2019-08-27-deployment-tooling-dpl-v2-preview-release) under Cleaning up the Git working directory.

skip_cleanup has been deprecated to cleanup. But cleanup is False by default which is the same behavior as skip_cleanup=True.

So I think removing it is fine. Thoughts?

Comment (vfdev-5) [reply .travis.yml]: Okay for me too. Thanks

**Comment by vfdev-5** [comment]
@anmolsjoshi seems like there is a config problem, as we can not deploy the docs now
https://travis-ci.org/pytorch/ignite/jobs/653819537#L3516

**Comment by vfdev-5** [comment]
Maybe `edge: true` is missing according to https://docs.travis-ci.com/user/deployment-v2#how-to-opt-in-to-v2

**Pull request merged**